nats-logr.go

```diff
@@ -0,0 +1,93 @@
+package natslogr
+
+import (
+	"time"
+
+	"github.com/go-logr/logr"
+	stan "github.com/nats-io/stan.go"
+	natslog "gomodules.xyz/nats-logr/nats-log"
```

Comment:
We should extract the necessary bits from `natslog` and use it directly here. We don't need / want that inner packgae. 

@masudur-rahman , let's talk tomorrow.